yourgov/components/FormMobileFoodVendorPermit/ApplicationHistory.tsx

```diff
@@ -50,7 +50,7 @@ export function ApplicationHistory() {
 					<TableHead>
 						<TableRow>
 							<TableHeader scope="col" width="18%">
-								Reference number
+								Reference&nbsp;number
```

Comment:
Interesting. Why did you do this?

Reply:
I didn't want them to ever wrap. My table uses percentages for its width